### ISSUE TITLE
Sensitivity with not trained words

### ISSUE THREAD

**Issue by ctrado18** [RasaHQ/rasa_nlu#1180]
Hey,

I face the strong issue that not trained words are not recognized very well (german).

If I test well trained sentences with unknown words NER fails, whereas with trained words it works...

That is actually really bad, since I want to have the entity from context and not train all possible words?!

What params with NER_CRF I have to tune or which features I should omit to overcome this high sensitivity for the words alone?

**Comment by ctrado18** [comment]
I thought so. I now got the feeling that training same sentences with multiple different entity values (using Chatito to substitutute a list of words for every sentence type) is not a good idea when using all features available?  When you use a bunch of words it might be focusing too much on the words itself and might overfit? So unknown words might be not detected. 

Is there a way how to find out if your model is overfitting/how well is your training data?

**Comment by amn41** [comment]
you can use `rasa_nlu.evaluate` with a test set http://nlu.rasa.com/evaluation.html#entity-extraction
